File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171036/Entities/AspNetUser.cs
using System;

namespace Models
{
	public class AspNetUsers
	{
	    public String Id { get; set; }
    public Guid PersonaId { get; set; }
    public bool IsDisable { get; set; }
    public String Email { get; set; }
    public bool EmailConfirmed { get; set; }
    public String PasswordHash { get; set; }
    public String SecurityStamp { get; set; }
    public String PhoneNumber { get; set; }
    public bool PhoneNumberConfirmed { get; set; }
    public bool TwoFactorEnabled { get; set; }
    public DateTime LockoutEndDateUtc { get; set; }
    public bool LockoutEnabled { get; set; }
    public int AccessFailedCount { get; set; }
    public String UserName { get; set; }

}
}

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Entities/AspNetUser.vb
Imports System

Public Class AspNetUsers
        Public Property Id As String
        Public Property PersonaId As Guid
        Public Property IsDisable As bool
        Public Property Email As String
        Public Property EmailConfirmed As bool
        Public Property PasswordHash As String
        Public Property SecurityStamp As String
        Public Property PhoneNumber As String
        Public Property PhoneNumberConfirmed As bool
        Public Property TwoFactorEnabled As bool
        Public Property LockoutEndDateUtc As DateTime
        Public Property LockoutEnabled As bool
        Public Property AccessFailedCount As int
        Public Property UserName As String

End Class